variables.tf

```diff
@@ -243,9 +243,12 @@ variable "cors" {
 }
 
 variable "website" {
-  type        = map(any)
+  type = map(object({
```

Comment:
IIUC now we will specify a map of bucket-names to configuration object? Could we provide a special key like `all` to ease the transition for users who want to use historical behavior? It would also be good to capture this in an upgrade guide https://github.com/terraform-google-modules/terraform-google-cloud-storage/tree/master/docs

Reply:
@bharathkkb If I'm not mistaken, if we use `all` it could break depending on the configuration the user enters.